modules.d/95udev-rules/module-setup.sh

```diff
@@ -56,7 +55,11 @@ install() {
     # eudev rules
     inst_rules 80-drivers-modprobe.rules
     # legacy persistent network device name rules
-    [[ $hostonly ]] && inst_rules 70-persistent-net.rules
+    if dracut_module_included "network"; then
+        [[ -e /etc/udev/rules.d/70-persistent-net.rules ]] && rm -f "$systemdnetwork"/99-default.link
```

Comment:
For `049`, you need to replace `$systemdnetwork` with `$systemdutildir/network`, because that variable didn't exist yet in that version. For `055` onwards it's ok.